docs/scripts/schemas/unversioned/eas-json-build-common-schema.js

```diff
@@ -59,11 +59,11 @@ export default [
     ],
   },
   {
-    name: 'buildArtifactsPaths',
+    name: 'buildartifactPaths',
```

Comment:
There's a typo here. The "a" should be capitalised :)

Reply:
Good catch